assets/admin/students/student-bulk-action-button/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { render, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import StudentModal from '../student-modal';
+
+/**
+ * Student bulk action button.
+ */
+export const StudentsBulkActionButton = () => {
+	const [ action, setAction ] = useState( 'add' );
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+	const [ studentIds, setStudentIds ] = useState( [] );
+	const closeModal = () => setIsModalOpen( false );
+	const setActionValue = ( selectedValue ) => {
+		switch ( selectedValue ) {
+			case 'enrol_restore_enrolment':
+				setAction( 'add' );
+				break;
+			case 'remove_enrolment':
+				setAction( 'remove' );
+				break;
+			case 'remove_progress':
+				setAction( 'reset-progress' );
+				break;
+			default:
+		}
+	};
+
+	const openModal = () => {
+		const hiddenSenseiBulkAction = document.getElementById(
+			'bulk-action-selector-top'
+		);
+		const hiddenSelectedUserIdsField = document.getElementById(
```

Comment:
This is part of our React component, so there's no need to use `document.getElementById`. We should be able to get its value directly if we set it appropriately.

Reply:
We can leave this as-is as per https://github.com/Automattic/sensei/pull/5004#discussion_r852328859.